book_source/05_developer_guide/03_git/01_Using-Git.Rmd

```diff
@@ -51,56 +40,69 @@ it. The git protocol is read-only.
  * git@github.com:PecanProject/bety.git
 * These instructions apply to other repositories too.
 
+#### PEcAn Project Branches
+
+We follow branch organization laid out on this page: http://nvie.com/posts/a-successful-git-branching-model](http://nvie.com/posts/a-successful-git-branching-model)/
+
+In short, there are three main branches you must be aware of:
+Develop - Main Branch conataining the latest code
```

Comment:
typo on containing. Also, you should expand the explanation in this section more -- at a minimum you should tell people that they will typically be committing into develop, they will never be committing into master, and they will only be committing into release when submitting fixes to a version that's being tested for release